CHANGELOG.md

```diff
@@ -1,19 +1,35 @@
+## 7.0.0
```

Comment:
Interestingly, github's preview mode doesn't show anything after 6.0.0.. must be a github bug?